src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md

```diff
@@ -50,6 +50,28 @@ Create an `blocklist.json` file with the following JSON content:
 }
 ```
 
+## Sample Block By Country Code (A two-character ISO 3166-1 country code for the country associated with the IP address.)
```

Comment:
```suggestion
**Sample Block By Country Code (A two-character ISO 3166-1 country code for the country associated with the IP address.):**

```

Reply:
There is a need to ban access through the country code.